modules/distr-tracing-tempo-config-query-rbac.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-configuring-query-rbac_{context}"]
+= Configuring the query RBAC
+
+As an administrator, you can set up the query role-based access control (RBAC) to filter the span attributes for your users by the namespaces for which you granted them permissions.
```

Comment:
Nit:

```suggestion
As an administrator, you can set up the query role-based access control (RBAC) to filter the span attributes for your users by the namespaces for which you granted them permissions.
```
```suggestion
As an administrator, you can set up the query role-based access control (RBAC) rules to filter the span attributes for your users in the namespaces where you granted their permissions.
```
IDK if that changes the intended meaning. (RBAC rules can get complicated and I find them difficult to write about. Sometimes we grant permissions so indirectly that what makes sense in YAML is difficult to translate into a sentence. At least for me.)

Reply:
Thank you for the suggestion, Michael. I have considered it. Indeed, I have found this to be a tough topic to handle.